File: teste.js
let salario_fixo = parseFloat(prompt("Digite o salário fixo do vendedor: salario_fixo"));
let vendas_efetuadas = parseFloat(prompt("Digite o valor total de vendas efetuadas: vendas_efetuadas"));
let premio = 0;

if (vendas_efetuadas > 7500) {
    premio = 1000;
} else if (vendas_efetuadas) {
    premio = 700;
} else if (vendas_efetuadas > 1000) {
    premio = 500;
}

let salario_total = salario_fixo + premio;

console.log("O salário total do vendedor é de R$", salario_total);

File: teste.py
salario = float(input('Qual o seu salário fixo em reais: R$'))
venda_efetuada = float(input('Qual foi o total de vendas mensal em reais: R$'))

if(venda_efetuada > 1000 and venda_efetuada <= 5000):
    salario_premio = salario + 500
    print('Seu salário esse mês será de', salario_premio)

elif(venda_efetuada > 5000 and venda_efetuada <= 7500):
    salario_premio2 = salario + 700
    print('Seu salário esse mês será de', salario_premio2)

elif(venda_efetuada > 7500):
    salario_premio3 = salario + 1000
    print('Seu salário esse mês será de', salario_premio3)

else:
    print('Seu salário será o mesmo!')